internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -252,41 +252,41 @@ func (a ApplicationSnapshotImage) ValidateAttestationSyntax(ctx context.Context)
 		return errors.New("no attestation data")
 	}
 
-	allErrors := map[string][]jsonschema.KeyError{}
+	var validationErr *jsonschema.ValidationError
 	for _, sp := range a.attestations {
 		pt := sp.PredicateType()
 		if schema, ok := attestationSchemas[pt]; ok {
 			// Found a validator for this predicate type so let's use it
 			log.Debugf("Attempting to validate an attestation with predicateType %s", pt)
-			if errs, err := schema.ValidateBytes(ctx, sp.Statement()); err != nil {
-				// Error while trying to validate
+
+			var st any
```

Comment:
Perhaps a comment to explain what `st` is, for code maintainability?

Reply:
Renamed to `statement`